prime-router/docs/openapi.yml

```diff
@@ -9,6 +9,160 @@ info:
   version: 0.2.0-oas3
 paths:
 
+  # The sender end-point handles receive actions.
+  #
+  # END-POINTS
+  # /submissions
+  # /submissions/submission/{reportid}
```

Comment:
Isn't this `actionId` instead of `reportId`?

Reply:
@RickHawesUSDS my thinking was that we use the `actionId` to generate a list of submissions, but point to the `reportId` when you move into an individual report. But, I can see the value in clicking into a single action so that we can display more of the data from the action, as well as the full data from the report.

Reply:
Oh, never mind, looking at the full file now, yes that's `actionId`. Will get these changes up in a minute.